code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const ThoughtForm = ({ newThought, onNewThoughtChange, onFormSubmit, onThoughtLikeChange }) => {
+  return (
+    <form
+      onSubmit={onFormSubmit}
+      className="thought-form">
+      <h1>Happy ❤️ Thought</h1>
+      <div className="post-thought">
+        <p>What&apos;s making you happy right now?</p>
+        <textarea
+          value={newThought}
+          onChange={onNewThoughtChange}
+          className="thought"
+          placeholder="write something nice here..." />
+        <p
+          className="character-counter">
+          {140 - newThought.length} / 140
```

Comment:
Nice solution for the character count. I had implemented this as well but this was a much simpler solution for the same functionality, so kudos!